src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpResponseStream.cs

```diff
@@ -72,7 +76,7 @@ public override void Flush()
             // Nothing to do.
         }
 
-        public override int Read(byte[] buffer, int offset, int count)
+        public override Task<int> ReadAsync(byte[] buffer, int offset, int count, CancellationToken token)
```

Comment:
As long as you're overriding Flush above, consider overriding FlushAsync as well:
```C#
public override Task FlushAsync(CancellationToken cancellationToken)
{
    return cancellationToken.IsCancellationRequested ?
        Task.FromCanceled(token) :
        Task.CompletedTask;
}
```